python-test/features/steps/agentsProvider.py

```diff
@@ -0,0 +1,54 @@
+from behave import given, when, then
+import time
+import functions
+import os
+import docker
+
+email = os.getenv('EMAIL')
+password = os.getenv('PASSWORD')
+
+agentname = os.getenv('AGENT_NAME')
+agentTagKey = os.getenv('TAG_KEY', 'test')
+agentTagValeu = os.getenv('TAG_VALUE', 'true')
+
+
+@given('A valid authentication')
+def get_auth(context):
+    context.token = functions.generate_token(email, password)
+    assert len(context.token) > 0
+
+@when('Create an agent')
+def create_agent(context):
+    context.status_agent, context.agent_data = functions.new_agent(context.token, agentname, agentTagKey, agentTagValeu)
+    assert context.status_agent == 201
+    context.agent_key = context.agent_data['key']
+    context.agent_id = context.agent_data['id']
+    context.agent_channel_id = context.agent_data['channel_id']
+
+@when('Run agent container')
+def run_agent(context):
+    iface = os.getenv('IFACE', 'mock')
+    context.orb_address = os.getenv('ORB_ADDRESS', 'beta.orb.live')
+    context.version_image = os.getenv('AGENT_VERSION')
+    context.agent_image = f"ns1labs/orb-agent:{context.version_image}"
```

Comment:
We shouldn't include the version field in the image if it was not provided.
In that case, `agent_image` would be simply `"ns1labs/orb-agent"` without the `:`.

Reply:
I don't think that will be a problem, right @weyrick ?

Reply:
Well it's not a problem per se, but I'm just wondering if we will get the results we expect. The tag for `orb-agent` should probably match the tag/version of the environment we are testing against

Reply:
`latest` will always match the latest version from self-host release, so 0.9.0 right now. As `develop` progresses, it may have had changes that we need for the tests to succeed.